cactusbot/handlers/command.py

```diff
@@ -181,8 +181,15 @@ def sub_args(match):
 
         _packet.sub(self.ARGS_EXPR, sub_args)
 
+        username = ""
+
+        if "token" in data:
```

Comment:
Could be made into a one-liner: `username = data.get("token", data.get("username", ""))`, but not necessary.